### PR TITLE
Add empty alt text to images to fix linter errors and reenable `jsx-a11y/alt-text`

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds empty `alt=""` attributes to images to fix `alt-text` errors in the `jsx-a11y` linter that was added [here](https://github.com/code-dot-org/code-dot-org/pull/55358).

**Followup:** I added TODO comments above each `` element that will be reviewed and updated w/ descriptive text if needed. See related Jira Ticket: [A11Y-279](https://codedotorg.atlassian.net/browse/A11Y-279)

**Jira ticket:** [A11Y-278](https://codedotorg.atlassian.net/browse/A11Y-278)

### CODE REVIEW

**Review by Hamms — COMMENTED** [review]
Is it possible to add meaningful text to these? An empty alt text string should be reserved exclusively for images that are used solely for decoration; several of the affected elements here seem functional.

**Comment by kelbyhawn** [comment]
> Is it possible to add meaningful text to these? An empty alt text string should be reserved exclusively for images that are used solely for decoration; several of the affected elements here seem functional.

Yeah! If I come across any I'm not sure about I'll ping y'all for more context.

**Review by hannahbergam — APPROVED** [review]
Yay thank you!!! Can't wait to get these all updated

**Review by bethanyaconnor — APPROVED** [review]
🎉

File: apps/test/unit/templates/studioHomepages/CourseCardTest.js [hunk 1]
```diff
@@ -29,7 +29,11 @@ describe('CourseCard', () => {
     expect(
       wrapper.containsMatchingElement(
         <a href={courses[0].link}>
-          <img />
+          {
```
Comment (bethanyaconnor) [inline apps/test/unit/templates/studioHomepages/CourseCardTest.js:32]: nit: Curious if we want this comment in tests? If the corresponding component does have alt text added, this test will fail but I think that's okay. On the other hand, if someone decides that empty alt text is fine for the corresponding component, they'll likely forget to remove this comment.

**Pull request merged**